papermill/api.py

```diff
@@ -221,24 +221,54 @@ def display_output(self, name):
         if name not in outputs:
             raise PapermillException(
                 "Output Name '%s' is not available in this notebook.")
-        output = outputs[name]
-        ip_display(output.data, metadata=output.metadata, raw=True)
+
+        # Loop over elements of output
+        for output in outputs[name]:
```

Comment:
Was there a contract change here we missed? Seems like this would explode if `output[name]` is not an iterable element. Do we need to guard against both patterns (likely)?